app/assets/stylesheets/application.scss

```diff
@@ -2,6 +2,7 @@
 @import "config/fonts";
 @import "config/colors";
 @import "config/bootstrap_variables";
+@import "config/utilities";
```

Comment:
We need to comment this out for the app to serve! So far it's the only place it's needed, but worth keeping just in case.